hqmedia.upload_controller.js

```diff
@@ -286,6 +286,10 @@ function BaseHQMediaUploadController (uploader_name, marker, options) {
         if (!self.allowCloseDuringUpload) {
             self.allowClose = false;
         }
+        if (!self.isMultiFileUpload) {
+            var newExtension = '.' + self.filesInQueueUI[0].get('name').split('.').pop().toLowerCase();
```

Comment:
What if there is more than one file in the queue? Should this be updated to handle each of them?